web_src/css/repo.css

```diff
@@ -3074,46 +3074,80 @@ tbody.commit-list {
   }
 }
 
-.pr-status {
-  padding: 0 !important; /* To clear fomantic's padding on .ui.segment elements */
+.commit-status-list {
+  padding: 0 !important;
+  max-height: 195px; /* fit exactly 4 items */
+  overflow-x: hidden;
+  transition: max-height .2s;
+  border-bottom: none !important;
+}
+
+.pr-commit-status[data-toggled="false"] {
+  border-bottom: none !important;
+}
+
+.pr-commit-status[data-toggled="false"] .ui.attached.header {
+  transition-delay: .2s;
+  border-bottom-width: 0 !important;
+}
+
+.pr-commit-status[data-toggled="false"] .commit-status-list {
+  max-height: 0;
```

Comment:
`max-height: 0` means `hidden`?

Reply:
Yes, but it's used for the transition that "slides" it in/out.

Reply:
Could there be a comment for its purpose?

ps: does `height: 0` work?

Reply:
comment added in https://github.com/go-gitea/gitea/pull/26284/commits/37197698156ddf4ebb47df50b8137beb39c61c6e.

`height: 0` could work, but it would require at least 1 more CSS property than using the existing `max-height`. I find it easier to read with `max-height`.